howtos/diffs/scheduled-sampling.diff

```diff
@@ -123,16 +123,10 @@ index 1597079..77449b7 100644
          decode_batch(optimizer.target, 5)
    return optimizer.target
  
-diff --git a/examples/seq2seq/train_test.py b/examples/seq2seq/train_test.py
-index 5a8a24b..6df633e 100644
---- a/examples/seq2seq/train_test.py
-+++ b/examples/seq2seq/train_test.py
-@@ -102,7 +102,7 @@ class TrainTest(absltest.TestCase):
-       model = train.create_model()
-       optimizer = train.create_optimizer(model, 0.003)
-       optimizer, train_metrics = train.train_step(
--          optimizer, batch, nn.make_rng())
-+          optimizer, batch, 0.5, nn.make_rng())
+@@ -390,4 +402,4 @@ def main(_):
  
-     self.assertLessEqual(train_metrics['loss'], 5)
-     self.assertGreaterEqual(train_metrics['accuracy'], 0)
+ 
+ if __name__ == '__main__':
+-  app.run(main)
++  app.run(main)
+\ No newline at end of file
```

Comment:
Can we fix this (re-add the newline, or just remove this line from the diff file?) Otherwise this will appear in the HOWTO view

Reply:
Er actually you need to remove this whole block -- this part of the code shouldn't change at all in the HOWTO

Reply:
I removed it.